gpgs/src/java/com/defold/gpgs/GpgsJNI.java

```diff
@@ -508,17 +508,20 @@ public void onComplete(@NonNull Task<SnapshotMetadata> task) {
         });
     }
 
-    public String resolveConflict(String conflictId, int metadataId) {
+    public void resolveConflict(String conflictId, int metadataId) {
         Snapshot snapshot = mPlayerSnapshot;
         if (metadataId == SNAPSHOT_CONFLICTING) {
             snapshot = mConflictingSnapshot;
         }
-        if (mPlayerSnapshot != null) {
-            mPlayerSnapshotsClient.resolveConflict(conflictId, snapshot)
-               .addOnCompleteListener(getOnLoadCompleteListener());
-            return null;
+        if (mPlayerSnapshot == null) {
+            sendSimpleMessage(SNAPSHOT_CONFLICTING,
+                    "status", STATUS_FAILED,
+                    "error",
+                    "Failed to resolve conflict. You aren't logged in.");
+            return;
         }
-        return "Failed to resolve conflict. You aren't logged in.";
+        mPlayerSnapshotsClient.resolveConflict(conflictId, snapshot)
```

Comment:
Not sure what kind of convention we have for long lines? Do we break long lines like that in other extensions?

Reply:
I use default Idea code formatter for java code